src/datepickerPopup/test/popup.spec.js

```diff
@@ -34,7 +34,7 @@ describe('datepicker popup', function() {
     var els = getLabelsRow().find('th'),
         labels = [];
     for (var i = dayMode ? 1 : 0, n = els.length; i < n; i++) {
-      labels.push(els.eq(i).text());
+      labels.push(els.eq(i).text().trim());
```

Comment:
Date strings weren't brin trimmed and caused a whole bunch of UTs to fail when i first cloned the repo. This addresses that issue.